draft-kazuho-httpbis-priority.md

```diff
@@ -300,6 +302,84 @@ receipt, a client that supports header-based prioritization MUST close the
 connection with a protocol error. Non-supporting clients will ignore this
 extension element (see {{!RFC7540}}, Section 5.5).
 
+# Reprioritization
+
+Once a client sends a request with a priority, circumstances might change and
+mean that it is beneficial to change the priority of the response. As an
+example, a web browser might issue a prefetch request for an HTML document with
+the urgency parameter of the Priority request header field set to `background`.
+Then, when the user navigates to the HTML while prefetch is in action, it would
+send a reprioritization frame with the priority field value set to `urgency=0`.
+
+However, a client cannot reprioritize a response by using the Priority header
+field.  This is because an HTTP header field can only be sent as part of an HTTP
+message. Therefore, to support reprioritization, it is necessary to define a
+HTTP-version-dependent mechanism for transmitting the priority parameters.
+
+This document specifies a new frame type for HTTP/2 ({{?RFC7540}}) and HTTP/3
+({{?I-D.draft-ietf-quic-http-23}}) that is specialized for reprioritization. It
+carries updated priority parameters and is sent on the version-specific control
+stream. The frame references the response to reprioritize based on a
+version-specific identifier; in HTTP/2 this is the Stream ID, in HTTP/3 this is
+either the Stream ID or Push ID.
+
+## HTTP/2 PRIORITY_UPDATE Frame
+
+The HTTP/2 PRIORITY_UPDATE frame (type=0xF) carries the stream ID of the
+response that is being reprioritized, and the updated priority in ASCII text,
+using the same representation as that of the Priority header field value.
+
+The PRIORITY_UPDATE frame is sent on stream 0.
```

Comment:
"The stream identifier for a PRIORITY_UPDATE frame MUST be zero (0x0)."

Reply:
fixed

Reply:
I realised this might be a little confusing because the H2 frame header includes the Stream Identifier field, and the frame payload contains a Stream ID field. So in https://github.com/kazuho/draft-kazuho-httpbis-priority/pull/51/commits/34600ed80d5242a536b6b16b403337dce85d3a77 I made it explicit.